App/src/pages/ManageableGroups.tsx

```diff
@@ -0,0 +1,98 @@
+import { Container, Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+import Button from "../components/Button";
+import { Link } from "react-router-dom";
+import { GetAllGroups } from "../services/group";
+import { GroupDTO, MultiGroupResponseDTO } from "../services/group";
+import { useEffect, useState } from "react";
+import { BaseRoutes } from "../constants";
+import Header from "../shared/Header";
+
+export default function DisplayManageableGroups() {
```

Comment:
This seems to display correctly, but when I try to click on the remove button for any group, the app crashes

Reply:
That's because they don't have any members, I wanted to have it limit the remove button to only when a group has members